docs/admin/_kubeadm/kubeadm.md

```diff
@@ -0,0 +1,60 @@
+## kubeadm
+
+kubeadm: easily bootstrap a secure Kubernetes cluster
+
+### Synopsis
+
+
+
+kubeadm: easily bootstrap a secure Kubernetes cluster.
+
+    ┌──────────────────────────────────────────────────────────┐
+    │ KUBEADM IS BETA, DO NOT USE IT FOR PRODUCTION CLUSTERS!  │
+    │                                                          │
+    │ But, please try it out! Give us feedback at:             │
+    │ https://github.com/kubernetes/kubeadm/issues             │
+    │ and at-mention @kubernetes/sig-cluster-lifecycle-bugs    │
+    │ or @kubernetes/sig-cluster-lifecycle-feature-requests    │
+    └──────────────────────────────────────────────────────────┘
+
+Example usage:
+
+    Create a two-machine cluster with one master (which controls the cluster),
+    and one node (where your workloads, like Pods and Deployments run).
+
+    ┌──────────────────────────────────────────────────────────┐
+    │ On the first machine                                     │
+    ├──────────────────────────────────────────────────────────┤
+    │ master# kubeadm init                                     │
+    └──────────────────────────────────────────────────────────┘
+
+    ┌──────────────────────────────────────────────────────────┐
+    │ On the second machine                                    │
+    ├──────────────────────────────────────────────────────────┤
+    │ node# kubeadm join <arguments-returned-from-init>        │
+    └──────────────────────────────────────────────────────────┘
+
+    You can then repeat the second step on as many other machines as you like.
+
+
+
+### Options
+
+```
+      --azure-container-registry-config string   Path to the file container Azure container registry configuration information.
```

Comment:
gah globals. We do hide these in the actual binary... is there any way we could hide them there as well @fabriziopandini?

Reply:
I think I managed to do this in a clean way. I will send a new PR for improving the `cmd\genkubedocs` in kubernets repo asap. In the meantime I updated all the generated files here